### ISSUE TITLE
Wifi samba share

### ISSUE THREAD

**Issue by milika** [pi1541/Pi1541#18]
Would be great if 1541 folder would be shared via wifi. No need to take out the sd card.

**Comment by alexanderk23** [comment]
Indeed, but it's a bare metal code -- there is no TCP/IP stack, no WiFi driver, no Linux kernel, no nothing. And barely just enough processing power to emulate 1541 itself, not to mention the bells and whistles.

**Comment by penfold42** [comment]
It should be doable.
There’s plenty of compute capacity - it’s quad core and 1 core can do all the hard emulation stuff. 
Lwip can provide the IP stack and from memory the USB stack has support for the network interface.

**Comment by milika** [comment]
would be great, depends probably on time most of all

**Comment by pottendo** [comment]
hi, pottendo-Pi1541 exactly implements networking support since quite a while - see here: https://github.com/pottendo/pottendo-Pi1541
bye, pottendo
